### PR TITLE
Install and configure release-it

### PR DESCRIPTION
**Pull request by lkostrowski**
Using setup from saleor dashboard.

### CODE REVIEW

File: .release-it.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "github": {
+    "release": true,
+    "draft": true,
+    "releaseName": "${version}"
+  },
+  "npm": {
+    "publish": false
```
Comment (krzysztofwolski) [inline .release-it.json:8]: Shouldn't it be set to true?

Comment (lkostrowski) [reply .release-it.json:8]: Probably 🤔 In Dashboard it's like this. I think they open draft release and a Github action is publishing

In this release, I will set `publish: true` and we will improve

Comment (krzysztofwolski) [reply .release-it.json]: Dashboard does not publish packages to npm (its a standalone app)

**Review by krzysztofwolski — APPROVED** [review]

**Comment by lkostrowski** [comment]
@krzysztofwolski leaving to you for improvement

**Pull request merged**